lib/screen/main/main_item.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:flutter/material.dart';
+import 'package:showcase_flutter_build_web_desktop/extension/context_extension.dart';
+import 'package:showcase_flutter_build_web_desktop/gen/colors.gen.dart';
+
+import '../../theme/dimen.dart';
+
+class MainItem extends StatelessWidget {
+  const MainItem({super.key});
+
+  // TODO integrate with real data
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: const BoxDecoration(
+        borderRadius: BorderRadius.all(Radius.circular(Dimens.borderRadius10)),
+        color: Colors.white,
+      ),
+      child: Padding(
+        padding: const EdgeInsets.all(Dimens.space10),
+        child: Column(
+          children: [
+            SizedBox(
+              width: double.infinity,
+              height: Dimens.mainItemHeight,
+              child: Container(
+                decoration: BoxDecoration(
+                  border: Border.all(
+                    color: ColorName.whitePlatinumColor,
+                  ),
+                  borderRadius: const BorderRadius.all(
+                      Radius.circular(Dimens.borderRadius5)),
+                ),
+                child: Image.network(
+                    'https://ucarecdn.com/7afb0e3c-7e2e-4f1a-8042-7cc86dbe0a48/'),
+              ),
```

Comment:
So, we don't need to define ` width: double.infinity,` ☝️ 

```suggestion
            Container(
              decoration: BoxDecoration(
                border: Border.all(
                  color: Colors.grey.shade100,
                ),
                borderRadius: const BorderRadius.all(
                  Radius.circular(Dimens.borderRadius5),
                ),
              ),
              child: Image.network(
                'https://ucarecdn.com/7afb0e3c-7e2e-4f1a-8042-7cc86dbe0a48/',
                height: 182, TODO // Rename dimen
              ),
```